packages/frontend/src/Build.svelte

```diff
@@ -27,6 +27,8 @@ import { goToDiskImages } from './lib/navigation';
 export let imageName: string | undefined = undefined;
 export let imageTag: string | undefined = undefined;
 
+const bootcExtensionGithubUrl = 'https://github.com/podman-desktop/extension-bootc';
```

Comment:
This could use the "repository" info from package.json (does not exist yet but could be added).

Reply:
https://github.com/podman-desktop/extension-bootc/pull/1835